src/Microsoft.ML.Core/Data/ITransformModel.cs

```diff
@@ -18,13 +18,20 @@ public interface ITransformModel
         /// Note that the schema may have columns that aren't needed by this transform model.
         /// If an IDataView exists with this schema, then applying this transform model to it
         /// shouldn't fail because of column type issues.
-        /// REVIEW: Would be nice to be able to trim this to the minimum needed somehow. Note
-        /// however that doing so may cause issues for composing transform models. For example,
-        /// if transform model A needs column X and model B needs Y, that is NOT produced by A,
-        /// then trimming A's input schema would cause composition to fail.
         /// </summary>
+        // REVIEW: Would be nice to be able to trim this to the minimum needed somehow. Note
+        // however that doing so may cause issues for composing transform models. For example,
```

Comment:
Ahh good. I'm half tempted to compose a regular expression to see how many of these beauties had slipped into the XML docs. :P :D